### ISSUE TITLE
Create a strategy for integrated approach with core Firefox OS Developer Participation Team

### ISSUE THREAD

**Issue by brianking** [mozilla/participation-org#96]
### Goal:

Pull together all the stakeholders into an integrated team, and create a strategy for participation in core Firefox OS development

**Planning document is here:**
https://docs.google.com/a/mozilla.com/document/d/1JBRVlUPA-L4efLVD0rxMcjwCgx_hglImHMu5_j_BQVs/edit?usp=sharing
NOTE: The scope of the plan does not currently include this, but will be expanded to do so.

**Objective:** Plan the basics of a new Firefox OS core development program

**Key result 1:** All stakeholders are on board and agree to work as a team

**Key result 2:** Weekly meetings

**Key result 3:** Draft strategy document
### Success:

1) Consult with core Firefox OS developers and bring them on board to help build out and execute the program.

2) Build a strategy/concept document on what core Firefox OS development participation looks like, with solid program recommendations. 70% quality -- draft version

3) Awareness of the new plan is distributed among the whole Firefox OS team
### Roles:

Owner: @brianking
Responsible: @brianking, @williamq
Involved: @autonome, @mikehenrty  
Reviewer: @autonome  
### Required:
- [x] Weekly meeting setup and first meeting takes place (@brianking 1 hr, Mon Aug 17)
- [x] Research on scope of the program (@brianking 2hrs, Weds Aug 12)
- [ ] Draft document with plan outline (@brianking 5hrs, Fri Aug 14)

**Comment by brianking** [comment]
The gaia / core dev team setup their own weekly meetings independent of what we are doing. We are attending and monitoring closely. However, we are more focused on foxfooding, b2gDroid, and porting so it is unsure if we will be integrating this into our plan. If we do, we will resurrect this in subsequent heartbeats.

In general, we are working very closely with key individuals on a holistic Firefox OS participation plan.

**Comment by LucyeoH-zz** [comment]
Draft document with plan outline - premature needs to be pushed back to end of next heartbeat (possibly already being done by gaia)  
Blocked by a lot of PTO - lots of meetings post-poned
